planetarium/NineChronicles
# Hammerpoint initialize when on changed avatar, Refactor HammerPointStates

### Description

If you change avatar, initializing hammer point too.

### How to test

1. Make any equipment.
2. Change avatar with 'Select Avatar' menu.
3. Check hammer point of equipment.

It already check with above flow.

### Related Links

[아바타 교체시 슈퍼크래프트 상태 갱신 싱크 맞추기](https://app.asana.com/0/958521740385861/1202962354804231/f)

### Required Reviewers

@planetarium/9c-dev
